app/app.go

```diff
@@ -1578,5 +1578,5 @@ func (app *App) decorateContextWithDexMemState(base context.Context) context.Con
 
 func init() {
 	// override max wasm size to 1MB
-	wasmtypes.MaxWasmSize = 1024 * 1024
+	wasmtypes.MaxWasmSize = 1280 * 1024
```

Comment:
any particular reason to choose this number vs. maybe just 2MB? also the comment above needs to be updated

Reply:
I'm inclined not to bump the uploadable wasm size too much as larger wasm file has negative impact on chain performance. Choosing this number to unblock Levana. We can also make a chain upgrade to adjust the param here, but let me know if you have a different thought

Reply:
will bump it to 2MB for now